<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabSimulate</class>
 <widget class="QWidget" name="TabSimulate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulate</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridSimulate">
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtSimLength">
       <property name="whatsThis">
        <string>&lt;h4&gt;Simulation length&lt;/h4&gt;
&lt;p&gt;The length of time in seconds that should be simulated&lt;/p&gt;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>&amp;Export...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QSlider" name="hslideTime">
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lblSimInterval">
       <property name="text">
        <string>Image interval:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="btnDisplayOptions">
       <property name="text">
        <string>Display options...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblCurrentTime">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="txtSimInterval">
       <property name="whatsThis">
        <string>&lt;h4&gt;Image interval&lt;/h4&gt;
&lt;p&gt;The length of time in seconds between each concentration image&lt;/p&gt;
&lt;p&gt;For example, if the simulation length is 100, and the image interval is 5, a total of 20 images would be produced with an interval of 5s of model simulation between each&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="btnSliceImage">
       <property name="text">
        <string>Slice image...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblSimLength">
       <property name="text">
        <string>Simulation length:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" colspan="2">
      <widget class="QPushButton" name="btnSimulate">
       <property name="whatsThis">
        <string>Run a simulation of the specified length, using the timestep 'dt' for the numerical integration</string>
       </property>
       <property name="text">
        <string>Simulate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="btnResetSimulation">
       <property name="whatsThis">
        <string>Reset all concentrations to their initial values, clear all simulation data.</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtSimLength</tabstop>
  <tabstop>txtSimInterval</tabstop>
  <tabstop>btnSimulate</tabstop>
  <tabstop>btnResetSimulation</tabstop>
  <tabstop>hslideTime</tabstop>
  <tabstop>btnSliceImage</tabstop>
  <tabstop>btnExport</tabstop>
  <tabstop>btnDisplayOptions</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
